appdev/guides/dependencies.rst

```diff
@@ -5,41 +5,35 @@ If an app depends on a library that is not pre-installed in Ubuntu Touch, the ap
 
 Building
 --------
-Often libraries are available pre-built, So you need to decide whether you want your build setup to download the library source and compile it or download the pre-built version. The following sections describe the options with its pros and cons.
+Sometimes libraries are available pre-built, allowing you to skip the build step. Compiling your dependency is recommended though, as it gives you more control over the whole process and may lead to better results (e.g. better performance, smaller package sizes etc.).
```

Comment:
Since you say "For example" in this sentence, there is no need for the additional "etc". It makes it unclear what exactly would be included in "etc" anyway.